getting_started/cloud_infrastructure_access/dedicated-aws-peering.adoc

```diff
@@ -0,0 +1,123 @@
+. Go to the *Peering Connections* page and click the *Create Peering Connection* button.
+. Double check details of the account you are logged in to and double-check the details of the account/VPC you are connecting to (from the first step above).
+.. Peering connection name tag - A descriptive name for the VPC Peering Connection.
+.. VPC (Requester) - This should be the {product-title} Cluster VPC ID, select from the dropdown.
+.. Account - Select *Another account*, and then provide the Customer AWS Account number (without dashes).
+.. Region - If the Customer VPC Region differs from the current region, select *Another Region* and select the customer VPC Region from the dropdown.
+.. VPC (Accepter) - The Customer VPC ID.
+. Click *Create Peering Connection*.
+. Confirm that the request enters a *Pending* state. If it enters a *Failed* state, confirm the details and repeat the process.
+
+.Procedure - Accept the VPC Peering Request
+Now that you have created the peering connection, you will need to accept the request in the Customer AWS Account.
+
+. Log into the AWS Web Console.
+. Navigate to *VPC Service*.
+. Go to *Peering Connections*.
+. Click on *Pending peering connection*.
+. Confirm the AWS Account and VPC ID that the request originated from. This should be from the {product-title} AWS Account and {product-title} Cluster VPC.
+. Click *Accept Request*.
+
+[id="peering-routing-tables"]
+== Configure the routing tables
+
+.Procedure
+When you accept the peering request, both VPCs will need to configure their routes to communicate across the peering connection.
+
+. Log into the AWS Web Console for the {product-title} AWS Account.
+. Go to the *VPC Service*, then *Route Tables*.
+. Select the Route Table for the {product-title} Cluster VPC.
++
+[NOTE]
+====
+On some clusters there may be more than one route table for a particular VPC. Select the private one that has a number of explicitly associated subnets.
+====
+
+. Select the *Routes* tab, then *Edit*.
+. Enter the Customer VPC CIDR block in the *Destination* text box.
+. Enter the Peering Connection ID in the *Target* text box.
+. Click *Save*.
+
+You will need to complete the same process with the other VPC’s CIDR block:
+
+. Log into the Customer AWS Web Console → *VPC Service* → *Route Tables*.
+. Select the Route Table for your VPC.
+. Select the *Routes* tab, then *Edit*.
+. Enter the {product-title} Cluster VPC CIDR block in the *Destination* text box.
+. Enter the Peering Connection ID in the *Target* text box.
+. Click *Save*.
+
+The VPC peering connection is now complete. Follow the steps in the verification section to ensure connectivity across the peering connection is working.
+
+[id="peering-verification-troubleshooting"]
+== Verification and Troubleshooting VPC Peering
```

Comment:
"Verifying and troubleshooting VPC peering"

Reply:
Though these should probably be two separate modules with separate procedures? Even if they're just single step/bullet.